packages/ui/src/lib/swr.ts

```diff
@@ -30,17 +31,17 @@ export const defaultFetcher = (params: string[] | string) => {
     resource = params
   }
   const headers = defaultHeaders(apiKey, clientVersion)
-  return fetch(resource, {
+  return axios(resource, {
     headers,
   })
     .then((res) => {
-      if (res.headers.get('deprecation') === 'true') {
+      if (res.headers?.['deprecation'] === 'true') {
         console.warn(
-          `Warning: API ${res.url} is deprecated. Stability and performance may be affected.`
+          `Warning: API ${res.config.url} is deprecated. Stability and performance may be affected.`
         )
       }
 
-      return res.json()
+      return JSON.parse(res.data())
```

Comment:
Something here was causing the `useTokens` hook on dev to return an empty array when there was actually token data being returned in the network tab. This temporarily broke the BuyModal